mmdet/models/plugins/dropblock.py

```diff
@@ -79,3 +79,7 @@ def _compute_gamma(self, feat_size):
         factor = (1.0 if self.iter_cnt > self.warmup_iters else self.iter_cnt /
                   self.warmup_iters)
         return gamma * factor
+
+    def extra_repr(self):
+        return (f'drop_prob={self.drop_prob}, block_size={self.block_size}, '
```

Comment:
This type of return (f'xxx') is not recommended, it is recommended to return 'xxx'